### PR TITLE
fix: src image

### PR DESCRIPTION
**Pull request by nguyendanghuydev**

### CODE REVIEW

File: src/components/Image/Image.tsx [hunk 1]
```diff
@@ -11,7 +12,11 @@ export type RImageProps = Omit<DetailedHTMLProps<ImgHTMLAttributes<HTMLImageElem
   };
 
 const RImage = ({ src, ...props }: RImageProps) => {
+<<<<<<< HEAD
```
Comment (jwthanh) [inline src/components/Image/Image.tsx:15]: @nguyendanghuydev 😮

**Comment by vkhanh74** [comment]
@nguyendanghuydev did you run `yarn storybook` to check edited component ?

**Comment by nguyendanghuydev** [comment]
yes, i did

File: src/components/Image/Image.tsx [hunk 2]
```diff
@@ -11,7 +12,7 @@ export type RImageProps = Omit<DetailedHTMLProps<ImgHTMLAttributes<HTMLImageElem
   };
 
 const RImage = ({ src, ...props }: RImageProps) => {
-  return <StyledImage src={!!src ? src : '@assets/images/placeholder.png'} {...props} />;
+  return <StyledImage src={!!src ? src : Img} {...props} />;
```
Comment (vkhanh74) [inline src/components/Image/Image.tsx:15]:
```suggestion
  return <StyledImage src={src || Img} {...props} />;
```

File: package.json [hunk 3]
```diff
@@ -34,8 +35,7 @@
     "rollup-plugin-postcss": "^4.0.2",
     "rollup-plugin-typescript2": "^0.31.1",
     "tsconfig-paths-webpack-plugin": "^3.5.2",
-    "typescript": "^4.5.4",
-    "next": "^12.0.8"
+    "typescript": "^4.5.4"
```
Comment (vkhanh74) [inline package.json:38]: Discard changes on this file

**Review by vkhanh74 — CHANGES_REQUESTED** [review]

**Pull request merged**